xds/service/orca/v3/orca.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
```

Comment:
I assume you'll do another pass where you flag in bold letters that the old UDPA version is frozen, stale etc?

Reply:
Done.